server/graphql/types/src/types/invoice_query.rs

```diff
@@ -282,11 +285,18 @@ impl InvoiceNode {
             None => patient_loader
                 .load_one(self.name_row().id.clone())
                 .await?
-                .map(|name_row| Name {
-                    name_row,
-                    name_store_join_row: None,
-                    store_row: None,
-                    properties: None,
+                .map(|name_row| {
+                    let name_id = name_row.id.clone();
+                    Name {
+                        name_row: name_row,
```

Comment:
Nit: var and initialisation same

```suggestion
                        name_row,
```

Reply:
[a791376](https://github.com/msupply-foundation/open-msupply/pull/7801/commits/a7913760a7de04a7bdcb30c046883a81b7bfa949)